pkgs/applications/radio/soapysdr/default.nix

```diff
@@ -17,6 +17,7 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "soapysdr";
+  # Don't forget to change passthru.abiVersion
   version = "0.8.2-pre";
 
   src = fetchFromGitHub {
```

Comment:
https://github.com/pothosware/SoapySDR/commit/309335ec6a52eb712387ed025d705a3c0f7a1e24

Maybe we should update the commit while we're at it.